manifests/certonly.pp

```diff
@@ -65,7 +65,7 @@
   }
   $command_end = inline_template('<% if @additional_args %> <%= @additional_args.join(" ") %><%end%>')
   $command = "${command_start}${command_domains}${command_end}"
-  $live_path = inline_template("${config_dir}/live/<%= @domains.first %>/cert.pem")
+  $live_path = inline_template("${config_dir}/live/<%= @domains.first.gsub('*.','') %>/cert.pem")
```

Comment:
Isn't gsub a regex? I'm wondering if this won't break on things. Unlikely, but would `*example.com` result in `xample.com`?

Reply:
Actually, I believe that if you give gsub a string as first argument, it doesn't behave like a regexp : 

```
irb(main):001:0> domains = [ '*example.com' ]
=> ["*example.com"]
irb(main):002:0> domains.first.gsub('*.','')
=> "*example.com"
irb(main):003:0> domains = [ '*.example.com' ]
=> ["*.example.com"]
irb(main):004:0> domains.first.gsub('*.','')
=> "example.com"
```

if you try to 'regexp' this first arg, you get an error : 

```
irb(main):004:0> domains.first.gsub(/*./,'')
Traceback (most recent call last):
SyntaxError ((irb):4: target of repeat operator is not specified: /*./)
```
;)

Reply:
From the [doc](https://ruby-doc.org/core-2.1.4/String.html#method-i-gsub) : 

```
The pattern is typically a Regexp; 
if given as a String, any regular expression metacharacters it contains will be interpreted literally, 
e.g. '\\d' will match a backlash followed by 'd', instead of a digit.
```

Nice to know :)